### ISSUE TITLE
About the dataset

### ISSUE THREAD

**Issue by wwangcece** [allenai/satlas-super-resolution#3]
Hello! Your work is excellent and has inspired and helped me a lot. However, this dataset seems to be too large, can you block it up and provide download links separately?

**Comment by piperwolters** [comment]
Hi there! Yes, I will start uploading individual sets today and hopefully you will be able to download them then.

**Comment by wwangcece** [comment]
Many thanks!

**Comment by wwangcece** [comment]
One more problem, I'm doing a large-factor super-resolution study, so I need some paired images with resolution of 32*32 and 512*512 to accomplish 16x super-resolution. I had been training and testing on synthetic data, and now I was eager to test my algorithms using real-world data. But I find that the data you publish seems to have been downsampled by a factor of four for NAIP. So can you provide NAIP data before downsampling? I would appreciate it if you can, and I will indicate the source of data in my research!

**Comment by piperwolters** [comment]
Ah, I see. I do have the original resolution NAIP data. It is obviously much larger than the downsampled version. Are you looking to train on this data or just test? If just test, I can quickly upload the val/test sets in full resolution.

**Comment by wwangcece** [comment]
Yes, in reality, I only need the validation set(8192 images). If possible, you can directly send the data to my email (2692809763@qq.com). Once again, express my gratitude for your open-source work!

**Comment by piperwolters** [comment]
Okay, awesome. I have uploaded the validation set in full resolution [here](https://pub-25c498004d1e4d4c8da69b2c05676836.r2.dev/satellite-super-resolution/val_set_full-res.tar). The directory structure is the same as the low-res version, except for all images are nested under '2019' and '2020' because the NAIP API recently changed and the image uuids are not the same as before.

And the urban train set is currently being uploaded as three parts. The first part is ready, [here](https://pub-25c498004d1e4d4c8da69b2c05676836.r2.dev/satellite-super-resolution/train_urban_set.z01).

Could you try downloading both of these to 1) make sure the validation set is how you expect it and 2) confirm that the new size of the training set chunks is small enough for you to download successfully?

**Comment by piperwolters** [comment]
Okay, awesome. I have uploaded the validation set in full resolution [here](https://pub-25c498004d1e4d4c8da69b2c05676836.r2.dev/val_set_full_res.tar). The directory structure is the same as the low-res version (naip/{image_uuid}/{zoom17_tile}/rgb.png), except for the image_uuid is now just '2019' or '2020' because the NAIP API recently changed the format in which files are downloaded.
And the urban train set is currently being uploaded as three parts. The first part is ready, [here](https://pub-25c498004d1e4d4c8da69b2c05676836.r2.dev/satellite-super-resolution/train_urban_set.z01).

Could you try downloading both of these to 1) make sure the validation set is how you expect it and 2) confirm that the new size of the training set chunks is small enough for you to download successfully?

**Comment by wwangcece** [comment]
Thank you very much, I can now download and use the validation and training sets as normal!
